### PR TITLE
added a note for the issue with libgdbm3

### LINKED ISSUE #12266: [Docs] libgdbm3 installation not working properly following instructions
### Current State
Following the instructions provided for prerequisites section of linux https://docs.meshery.io/project/contributing/contributing-docs#for-linux

There is an error for libgdbm3
```
(base) om@om:~$ sudo apt-get install autoconf bison build-essential libssl-dev libyaml-dev libreadline6-dev zlib1g-dev libncurses5-dev libffi-dev libgdbm3 libgdbm-dev
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Note, selecting 'libreadline-dev' instead of 'libreadline6-dev'
Note, selecting 'libncurses-dev' instead of 'libncurses5-dev'
E: Unable to locate package libgdbm3
```

### Desired State
Actually, latest versions of debian based os doesn't support libgdbm3

So depending upon the os version libgdbm5 or libgdbm6 can be used, following this https://github.com/rbenv/ruby-build/issues/1211

A note should we added after prerequisite section in the linux installation section of docs , explaining the above thing.

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by omgupta-iitk**
**Notes for Reviewers**

This PR fixes #12266

Added a note for the issue with libgdbm3.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 600cc8762e1048afe445df6066405efacfeb0cbb at: https://67016215f9b5332e0618d5be--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit edc127dd3876238a5b1a8c7cdcaff038670b5bfb at: https://67026f0c2e1e5f34b1409570--meshery-docs-preview.netlify.app

File: docs/pages/project/contributing/contributing-docs.md [hunk 1]
```diff
@@ -106,6 +106,7 @@ Meshery documentation is built using the following components:
   sudo apt-get install autoconf bison build-essential libssl-dev libyaml-dev libreadline6-dev zlib1g-dev libncurses5-dev libffi-dev libgdbm3 libgdbm-dev
   </code>
   </div></pre>
+{% include alert.html type="info" title="Note" content="Latest versions of debian based OS does not support libgdbm3, so if you encounter any issue with its installation then try replacing it with libgdbm6 or libgdbm5." %}
```
Comment (leecalcote) [inline docs/pages/project/contributing/contributing-docs.md:109]:
```suggestion
{% include alert.html type="info" title="Note" content="The latest versions of Debian-based OS'es do not support <code>libgdbm3</code>, so if you encounter an issue with it's installation, try replacing <code>libgdbm3</code> with <code>libgdbm5</code> or <code>libgdbm6</code>." %}
```

**Review by leecalcote — APPROVED** [review]
Looks great. Thank you, @omgupta-iitk 👍

**Pull request merged**